Cargo.toml

```
@@ -0,0 +1,4 @@
+[package]
+name = "literal-escaper"
+version = "0.0.1"
+edition = "2021"
```

```suggestion
edition = "2021"
description = "Provides code to unescape string literals"
license = "Apache-2.0 OR MIT"
repository = "https://github.com/rust-lang/literal-escaper"
```
we should also provide the licenses files in this repo as well